### PR TITLE
Anchor image download from backend

### PR DESCRIPTION
**Pull request by Wollaars**

### CODE REVIEW

**Review by IndyV — CHANGES_REQUESTED** [review]
App seems to crash sometimes (hard to replicate and sometimes does show good result).
After attempting to load a new manual I also get this 
image

**Comment by IndyV** [comment]
Can't replicate consistently. Seems that loading a manual with steps but 0 interactions makes the app behave weirdly

File: app/src/main/java/ml/moneo/app/activity/ManualActivity.kt [hunk 1]
```diff
@@ -1,31 +1,55 @@
...
+
+        manualViewModel.getManual().observe(this, {
+            manualViewModel.loadBitmap()
+        })
+
+        manualViewModel.getBitmap().observe(this, {
+            setContentView(R.layout.activity_main);
+            var deviceId = ""
+
+            Log.d("Logge", "Hibabe")
```
Comment (Wollaars) [inline app/src/main/java/ml/moneo/app/activity/ManualActivity.kt]:
```suggestion
```

File: app/src/main/java/ml/moneo/app/activity/ManualActivity.kt [hunk 2]
```diff
@@ -1,31 +1,55 @@
...
+
+            val fragment = supportFragmentManager.findFragmentById(R.id.included_manual) as ManualFragment
+            Log.d("Logge", fragment.toString())
```
Comment (Wollaars) [inline app/src/main/java/ml/moneo/app/activity/ManualActivity.kt]:
```suggestion
```

File: app/src/main/java/ml/moneo/app/activity/ManualActivity.kt [hunk 3]
```diff
@@ -1,31 +1,55 @@
...
+            intent.getStringExtra("MANUAL_ID")?.let { Log.d("Logge", it) }
```
Comment (Wollaars) [inline app/src/main/java/ml/moneo/app/activity/ManualActivity.kt]:
```suggestion
```

File: app/src/main/java/ml/moneo/app/activity/fragment/GuidesCatalogFragment.kt [hunk 4]
```diff
@@ -61,6 +62,7 @@ class GuidesCatalogFragment : CatalogFragment(R.layout.fragment_catalog_overview
 
         (activity as CatalogsOverviewActivity?)?.blockUserInput(true)
 
+        Log.d("Logge", "Hi?")
```
Comment (Wollaars) [inline app/src/main/java/ml/moneo/app/activity/fragment/GuidesCatalogFragment.kt]:
```suggestion

```

File: app/src/main/java/ml/moneo/app/viewmodel/ManualViewModel.kt [hunk 5]
```diff
@@ -39,6 +45,32 @@ class ManualViewModel : ViewModel() {
         return manual.value!!.steps[currentStep.value!!].interactions
     }
 
+    fun loadBitmap() {
+        try {
+            GlobalScope.launch{
+                Log.d("Tagge", getAnchorImageURL());
```
Comment (Wollaars) [inline app/src/main/java/ml/moneo/app/viewmodel/ManualViewModel.kt]:
```suggestion
```

**Review by houfio — APPROVED** [review]

**Review by IndyV — APPROVED** [review]

**Pull request merged**